src/main/java/org/prebid/server/bidder/sharethrough/model/UserExt.java

```diff
@@ -0,0 +1,19 @@
+package org.prebid.server.bidder.sharethrough.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import org.prebid.server.proto.openrtb.ext.request.ExtUserEid;
+
+import java.util.List;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
This is unnecessary as we set Vert.x ObjectMapper with configuration `FAIL_ON_UNKNOWN_PROPERTIES, false` which is basically the same. Check out org.prebid.server.json.ObjectMapperConfigurer.